cloud/azure/bin/setup-saml-keystore

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env bash
+
+# TODO: figure out where in command flow to export PROJECT_BASE_DIR to
+# make it available in this file
+PROJECT_BASE_DIR="$(realpath "$(dirname "${BASH_SOURCE}")/../../../")"
+source "${PROJECT_BASE_DIR}/cloud/azure/bin/lib.sh"
+
+# DOC: Generate PKCS12 keystore and upload to Azure Blob Storage. 
+# DOC: Make sure to run the key_vault_setup script before running this one.
+# DOC: Required flags are:
+# DOC: g - Azure resource group
+# DOC: v - Azure key vault name
+# DOC: s - Azure storage account name
+# DOC: l - Azure region
+
+while getopts g:v:s:l: flag
+do
+  case "${flag}" in
+    g) resource_group=${OPTARG};;
+    v) vault_name=${OPTARG};;
+    s) storage_account_name=${OPTARG};;
+    l) location=${OPTARG};;
+  esac
+done
+
+if [[ ! "${resource_group}" ]] \
+   || [[ ! "${storage_account_name}" ]] \
+   || [[ ! "${vault_name}" ]] \
+   || [[ ! "${location}" ]]; then
+  out::error "arguments -g, -s, v, and -l must be provided"
+  exit 1
+fi
+
+echo "Assign the role 'Storage Blob Data Contributor' to the current user"
+storage::assign_storage_blob_data_contributor_role_to_user "${resource_group}"
+
+echo "Create resource group ${resource_group}"
+azure::create_resource_group "${resource_group}" "${location}"
```

Comment:
does this add a 3rd resource group for the deploy

Reply:
only if you use a different resource group from the key vault. I've been using the same one for both

Reply:
alright yeah i guess with the deploy script we can just automate this aspect of it